config/locales/en.yml

```diff
@@ -137,8 +140,8 @@ en:
       initial_teacher_training_subject_specialism: "Which science subject did your initial teacher training specialise in?"
       has_uk_maths_or_physics_degree: "Do you have a UK undergraduate or postgraduate degree in maths or physics?"
       qts_award_years:
-        before_september_2014: "In or before the academic year 2013 to 2014"
-        on_or_after_september_2014: "In or after the academic year 2014 to 2015"
+        before_cut_off_date: "In or before the academic year 2013 to 2014"
+        on_or_after_cut_off_date: "In or after the academic year 2014 to 2015"
```

Comment:
Can we remove these policy specific translation strings now that we're using `t("answers.qts_award_years.on_or_after_cut_off_date", year: current_policy.first_eligible_qts_award_year.to_s(:long))` in the view?

Reply:
Not yet, at least not without breaking a bunch of specs. There are still a bunch of places the existing keys are being used, mostly in the specs. Part of the follow-up work to update admin so that it dynamically shows the correct academic year will also migrate to using the new keys everywhere. At that point we can ditch these old keys.